tests/Feature/FileUploaderTest.php

```diff
@@ -16,7 +16,7 @@ public function a_user_can_upload_file()
             'name'      => 'test',
             'path'      => $this->getTestFilesDirectory('test.psd'),
         ];
-        $file = new UploadedFile($data['path'], $data['name'], 'image/jpeg', filesize($data['path']), null, true);
+        $file = new UploadedFile($data['path'], $data['name'], 'image/jpeg', filesize($data['path']), false, true);
```

Comment:
@RichardDominik can you explain this change?

Reply:
@palypster the $size parameter was removed in Symfony UploadFile class  https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/File/UploadedFile.php